src/components/chatRooms/ChatRoomForm.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import TextInput from '../common/TextInput';
+import {Component} from 'react/lib/ReactBaseClasses';
+import ChatRoomApi from '../../api/chatRoom';
+
+class ChatRoomForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      chatRoom: {
+        name: ''
+      }
+    };
+
+    this.save = this.save.bind(this);
+    this.change = this.change.bind(this);
+  }
+
+  save(event) {
+    event.preventDefault();
+    ChatRoomApi.create(this.state.chatRoom);
```

Comment:
This is not very redux-friendly: in redux, everything should happen thru [actions](https://redux.js.org/docs/basics/Actions.html) and some components (the [containers](https://medium.com/@dan_abramov/smart-and-dumb-components-7ca2f9a7c7d0)) are [connected](https://redux.js.org/docs/basics/UsageWithReact.html) to actions.

Reply:
With redux, you should not use API directly but use actions instead